tools/sense_studio/annotation.py

```diff
@@ -87,7 +84,8 @@ def prepare_annotation(project):
             compute_frames_features(inference_engine=inference_engine,
                                     videos_dir=videos_dir,
                                     frames_dir=frames_dir,
-                                    features_dir=features_dir)
+                                    features_dir=features_dir,
+                                    with_features=False)
```

Comment:
Shouldn't `with_features` depend on the setting?

Reply:
Done.